deploy/components/vllm-sim-pd/deployments.yaml

```diff
@@ -49,7 +49,7 @@ spec:
     spec:
       initContainers:
       - name: routing-sidecar
-        image: ghcr.io/llm-d/llm-d-routing-sidecar:latest
+        image: ${ROUTING_SIDECAR_IMAGE}
```

Comment:
Please undo this change. The file as it was allows the YAML file to be used outside of the kind based tests.